src/state/recipes/actions.js

```diff
@@ -145,13 +145,30 @@ export function fetchRecipeHistory(pk) {
 
 export function fetchRecipeFilters() {
   return async dispatch => {
+    const localStorageKey = 'recipe_filters';
+    const localFilters = window.localStorage.getItem(localStorageKey);
+    if (localFilters) {
+      // If the filters *were* in localStorage, it's a JSON *string*.
+      dispatch({
+        type: RECIPE_FILTERS_RECEIVE,
+        filters: JSON.parse(localFilters),
+      });
+    }
+    // XXX If we already had the filters in localStorage, can we avoid using this
+    // makeNormandyApiRequest() with an ID so that it doesn't cause any spinners to appear
+    // when the localStorage version almost definitely is good enough.
```

Comment:
nit: can we address this?

Reply:
[Check this out](https://github.com/mozilla/delivery-console/pull/489/commits/4fe13f9e2d7f11ca2dc7f1708c543c1a00306640). In practice it works. 
Basically, if there *was* data in `localStorage` it makes the next XHR request *without* setting `{[requestId]: {inProgress:true}}`.

See this:
<img width="1616" alt="screen shot 2018-10-18 at 10 18 08 am" src="https://user-images.githubusercontent.com/26739/47162065-48ff1d80-d2c1-11e8-8930-ee8c11c2c91e.png">

I put in a `time.sleep(10)` in the `Filters(views.APIView).get` method. Note how all the fields are *not* disabled whilst waiting for this request. 

Then I delete the stuff from `localStorage` and refreshed. This time, all the fields are disabled whilst waiting. 
<img width="1616" alt="screen shot 2018-10-18 at 10 31 02 am" src="https://user-images.githubusercontent.com/26739/47162166-75b33500-d2c1-11e8-9448-a36cd48ea7b2.png">